README.md

```diff
@@ -87,13 +87,6 @@ to build in prod use command
 
  These have been added to pipeline so when you do a push, this should run automatically. you can check in docker to see your project, and also in the github repo to see jobs running
 
- install heroku curl https://cli-assets.heroku.com/install-ubuntu.sh | sh
- heroku login -i work email, password in secrets file
- heroku container:login
-
- change the tag from docker to heroku
- docker tag kelliep/my-todo-app:latest registry.heroku.com/to-do-app-kellie/web
- push to heroku
- docker push registry.heroku.com/to-do-app-kellie/web
- then release to Heroku
- heroku container:release -a to-do-app-kellie web
\ No newline at end of file
+ 
+
+ 
```

Comment:
Getting rid of the old info is great, but it would be nice to link the new website now! <https://kp01012022.azurewebsites.net/>